util/logutil/log_test.go

```diff
@@ -122,3 +122,28 @@ func (s *testLogSuite) TestSetLevel(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(log.GetLevel(), Equals, zap.DebugLevel)
 }
+
+func (s *testLogSuite) TestGrpcLoggerCreation(c *C) {
+	conf := NewLogConfig("info", DefaultLogFormat, "", EmptyFileLogConfig, false)
+	copiedConfig := *conf
+	// assert that the new conf's address not same as the old one
+	c.Assert(&copiedConfig != conf, Equals, true)
```

Comment:
how about aseerting that after calling the function the original config's level remains unchanged?

Reply:
fixed